README.md

```diff
@@ -42,13 +42,13 @@ To build ONNF, use the following command:
 
 [same-as-file]: <> ({"ref": "utils/install-onnf.sh", "skip-doc": 2})
 ```
-git clone --recursive git@github.com:clang-ykt/ONNF.git
+git clone --recursive git@github.com:<<<your fork>>>/onnx-mlir.git
```

Comment:
@tjingrant 
interesting: this fails because the test in install is different. How do you think we should handle this one? Install needs an actual fork? or we can just take the main repo, but add a comment that we should use our own fork in the readme?

Reply:
Hi Alex, I think this is a case where DocCheck is doing its job. The buildbot failure is caused by Line 51, which DocCheck detects is different from utils/install-onnf.sh. So you may want to sync L51 here with the corresponding line in utils/install-onnf.sh.

In this particular line where we clone the git repository, there shouldn't be a problem with cloning simply from git@github.com:onnx/onnx-mlir.git.